src/services/jwt.service.js

```diff
@@ -0,0 +1,48 @@
+import jwt from 'jsonwebtoken';
+import 'dotenv/config';
+
+function sign(user) {
+  const token = jwt.sign(user, process.env.JWT_KEY, {
+    expiresIn: '50s',
```

Comment:
The expiration time for the access token is set to 50 seconds, which is quite short. Consider increasing the expiration time to a more practical duration, such as 15 minutes or 1 hour, depending on your application's requirements.